src/Test/L1/Worker/L1TestBase.cs

```diff
@@ -108,21 +107,29 @@ protected static TaskStep CreateScriptTask(string script)
             var stringFile = Path.Combine(assemblyLocation, "en-US", "strings.json");
             StringUtil.LoadExternalLocalization(stringFile);
 
-            using (L1HostContext context = new L1HostContext("Agent", GetLogFile(this, testName)))
```

Comment:
We probably should still dispose the `L1HostContext` somewhere.

Reply:
Add disposal after tests.